executor/infoschema_reader.go

```diff
@@ -1848,11 +1849,24 @@ func (e *memtableRetriever) setDataForPlacementPolicy(ctx sessionctx.Context) er
 			continue
 		}
 		// Currently, only partitions have placement rules.
+		var tbName, dbName, ptName string
+		skip := false
```

Comment:
Maybe `true` by default?

Reply:
Whoops, yes. my thoughts did not flow smoothly today.